presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -66,8 +66,6 @@ public void testJoins()
     @Test
     public void testFailures()
     {
-        assertQueryFails("SELECT sequence(1, orderkey) FROM orders",
```

Comment:
This test is for verifying the behavior when a function doesn't exist. Instead of removing the test, would you change it to use a different function?